### PR TITLE
🎨 935 gradient for connector section

### PR DESCRIPTION
**Pull request by jasangill1**
## About 

-used an eclipse to make the radial gradient set stops at 50 percent and start at 5%

## Notes 

- I am complaining to Tailwind cuz that was so painful

Screen Shot 2023-07-02 at 7 49 12 PM

### CODE REVIEW

File: next/src/components/landing/Connector.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const Connector = () => {
```
Comment (awtkns) [inline next/src/components/landing/Connector.tsx:3]: Might want to rename this component to something more descriptive :)

File: next/src/components/landing/Connector.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  return (
+    <div className="flex h-screen w-screen items-center justify-center bg-[radial-gradient(ellipse_at_center,_var(--tw-gradient-stops))] from-slate-900 from-10% via-transparent via-70% to-transparent "></div>
```
Comment (awtkns) [inline next/src/components/landing/Connector.tsx:5]: Nice work here!

```suggestion
    <div className="flex h-screen w-screen items-center justify-center bg-[radial-gradient(ellipse_at_center,_var(--tw-gradient-stops))] from-slate-900 from-10% via-transparent via-70% to-transparent"></div>
```

**Review by awtkns — APPROVED** [review]

File: next/src/components/landing/Connector.tsx (hunk 1 above)
Comment (asim-shrestha) [reply next/src/components/landing/Connector.tsx:3]: ConnectorSection?

File: next/src/components/landing/Connector.tsx (hunk 2 above)
Comment (asim-shrestha) [reply next/src/components/landing/Connector.tsx:5]: Lets make it purple?

**Comment by jasangill1** [comment]
Screen Shot 2023-07-02 at 8 08 56 PM

**Comment by asim-shrestha** [comment]
image

**vs.** 

image

- Need a much softer purple
- Overflow is getting cut off

**Comment by jasangill1** [comment]
Screen Shot 2023-07-02 at 9 58 53 PM

**Comment by jasangill1** [comment]
Screen Shot 2023-07-02 at 10 02 14 PM
Screen Shot 2023-07-02 at 10 02 14 PM (2)

**Comment by awtkns** [comment]
Seems like you may need to play with contrast and saturation (tailwind css) to get the look you want.

**Review by awtkns — APPROVED** [review]

**Pull request merged**